source/Command.cpp

```diff
@@ -53,34 +53,33 @@ const Command Command::LEFT(ONE << 2, "Turn left");
 const Command Command::RIGHT(ONE << 3, "Turn right");
 const Command Command::BACK(ONE << 4, "Reverse");
 const Command Command::MOUSE_TURNING_HOLD(ONE << 5, "Mouse turning (hold)");
-const Command Command::MOUSE_TURNING_TOGGLE(ONE << 6, "Mouse turning (toggle)");
```

Comment:
After resolving conflicts I removed this one command and shifted all of the numbers.